docs/grafana-dashboard.md

```diff
@@ -0,0 +1,130 @@
+
+```
+NAME                                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+alertmanager-operated                          ClusterIP   None            <none>        9093/TCP,9094/TCP,9094/UDP   31m
+prometheus-operated                            ClusterIP   None            <none>        9090/TCP                     30m
+prometheus-operator-alertmanager               ClusterIP   10.102.104.48   <none>        9093/TCP                     31m
+prometheus-operator-grafana                    NodePort    10.96.160.172   <none>        80:31409/TCP                 31m
+prometheus-operator-kube-state-metrics         ClusterIP   10.105.92.154   <none>        8080/TCP                     31m
+prometheus-operator-operator                   ClusterIP   10.99.15.245    <none>        8080/TCP,443/TCP             31m
+prometheus-operator-prometheus                 ClusterIP   10.109.75.138   <none>        9090/TCP                     31m
+prometheus-operator-prometheus-node-exporter   ClusterIP   10.98.128.115   <none>        9100/TCP                     31m
+```
+
+## Accessing Grafana UI over SSH Tunnel
+
+### Windows using PuTTY
+
+- Get the Kubernetes Worker Node IP and the Grafana Node Port.
+- Get the SSH server using which, Kubernetes Worker Node IP is accessible. Say this is Landing IP.
+- Configure the PuTTY as follows:
+  - Create a new Session with Landing IP, Landing Port
+  - Create a Connection -> SSH -> Tunnels
+    - Source Port = Grafana NodePort
+    - Destination = Kubernetes Worker Node IP:Grafana Node Port
+  - Open the PuTTY session. Enter SSH user name and passowrd for the Landing IP.
+- Now you can access Grafana UI at http://localhost:<Grafana-Node-Port>/. Default login and password ( admin/prom-operator )
+
+
+### Linux using SSH
+
+- Get the Kubernetes Worker Node IP and the Grafana Node Port.
+- Get the SSH server using which, Kubernetes Worker Node IP is accessible. Say this is Landing IP.
+- Open SSH tunnel using command.
+  ```
+  ssh -NL <Grafana-Node-Port>:<k8s-worker-node-IP>:<Grafana-Node-Port> <landing-machine-user>@<landing-machine-ip> -p <landing-machine-ssh-port>
+  ```
+- Now you can access Grafana UI at http://localhost:<Grafana-Node-Port>/. Default login and password ( admin/prom-operator )
+
+## Verify Granafa Dashboard
+
+- Login to Granfa UI
+- Click on Settings -> Data Source. You must see a Default Prometheus data source for `http://prometheus-operator-prometheus:9090/`
+- Click on Dashboards -> Manage Dashboards. You must see a list of dashboards. Click on any of them like: `kubernetes-compute-resources-cluster`
+
+## Add ZFS Grafana Dashboard
+
+- Login to Granfa UI
+- Click on Create Dashboard -> Import dashboard
+- Paste the below json and Click on Load
+
+https://raw.githubusercontent.com/openebs/zfs-localpv/master/deploy/sample/grafana-dashboard.json
+
+- Select datasource as Prometheus and Import it.
+
+- Now you should see a Dashboard with name as "ZFS"
```

Comment:
n_: Can this be "ZFS Local PV", as it does have some metrics and ui elements around ZFS Local PV.

Reply:
updated the name to ZFS-LocalPV.